pages/checkout.vue

```diff
@@ -417,8 +413,41 @@ export default {
       await setShippingInfo(params)
     }
 
-    const savePaymentDetails = () => {
-      console.log("Saved payment details.")
+    const getPaymentMethodData = (updatedPaymentDetails) => {
+      cardDetails = {
+        ...updatedPaymentDetails,
+      }
+    }
+
+    const addPaymentMethod = async () => {
+      const creditCardData = {
+        cardNumber: cardDetails.value.cardNumber,
+        cardType: cardDetails.value.cardType,
+        // cardholderName: creditCardPaymentGetters.getNameOnCard(cardDetails.value),
+        cvv: cardDetails.value.securityCode,
+        expireMonth: creditCardPaymentGetters.getExpireMonth(cardDetails?.value),
+        expireYear: creditCardPaymentGetters.getExpireMonth(cardDetails?.value),
+      }
+
+      const tokenizedData = await tokenizeCard(creditCardData)
+
+      if (tokenizedData) {
+        const paymentAction = buildPaymentMethodInput(
```

Comment:
once billing info is complete, we need to pass it here in the buildPaymentMethodInput